src/main/java/cz/cesnet/cloud/occi/api/http/HTTPClient.java

```diff
@@ -424,6 +425,42 @@ public URI create(Entity entity) throws CommunicationException {
     }
 
     /**
+     * @see Client#update(cz.cesnet.cloud.occi.core.Entity)
+     */
+    @Override
+    public boolean update(Entity entity) throws CommunicationException {
+        Kind kind = entity.getKind();
+        if (kind == null) {
+            throw new CommunicationException("entity with empty kind");
+        }
+
+        HttpPut httpPut = HTTPHelper.preparePut(kind.getLocation() + entity.getId(), connection.getHeaders());
+        try {
+            switch (mediaType) {
+                case MediaType.TEXT_OCCI: {
+                    Headers headers = entity.toHeaders();
+                    addHeaders(httpPut, headers);
+                }
+                break;
+                case MediaType.TEXT_PLAIN: {
+                    HttpEntity httpEntity = new StringEntity(entity.toText());
+                    httpPut.setEntity(httpEntity);
+                }
+                break;
+                default:
+                    throw new CommunicationException("unsupported media type '" + mediaType + "'");
+            }
+
+            checkConnection();
+            runAndParseRequest(httpPut, HttpStatus.SC_OK);
+
+            return true;
```

Comment:
Can you instead of the method `runAndParseRequest` and returning `true` use method `runRequestForStatus` which can be found as static method on class `HTTPHelper`, as it's used in `delete` method? Parsed response is not used anyway and this way the boolean return value would make more sense.